contrib/docker/testnet.sh

```diff
@@ -31,10 +31,19 @@ fi
 
 Prepare()
 {
+  # Ensure we have proper binaries installed
+  hash docker 2>/dev/null
+  catch "zip not installed, please install"
```

Comment:
> zip

docker

Reply:
/me falcepalms

pre-rebased copypasta, thanks for catching.